### DISCUSSION TITLE
xml error?

### DISCUSSION THREAD

**Discussion by PPQQXX** [onvif/testspecs#46] [Q&A] [answered]
Version: ONVIF Device Test Tool version 22.12 rev. 28
Test Case ID: MEDIA-2-1-9
![image](https://github.com/onvif/testspecs/assets/32433430/3665cc1f-7d24-4894-9ed1-91f8606127c6)

i add log in front of send hello message andh behind,but it seems ok,deivce not print error message, but the error come out One percent of the time
.
I find if  i shutdown the device program when ODT was accepting hello message ,the error must will come out.

**Comment by madhurao68** [comment]
Hi,
We will take this up in our call with the vendor next week and keep you posted. Thanks for your patience!

**Comment by madhurao68** [comment, marked as answer]
Hello,
From the description it could be due to invalid XML from camera, or something else was sent to the multicast: address 239.255.255.250 port 3702.
Could you upload Wireshark logs (specific to multicast address 239.255.255.250 port 3702)? Thanks!
